tests/post/steps/test_volume.py

```diff
@@ -100,6 +110,31 @@ def _check_size():
         name='checking size of PersistentVolume {}'.format(name)
     )
 
+
+@then(parsers.parse("the Volume '{name}' does not exists"))
+def volume_not_exist(host, name, k8s_custom_client, k8s_client):
+    def _check_volume():
+        volume = _volume_exist(host, name, k8s_custom_client)
+        if volume is not None:
+            assert False, 'Volume {} still exist'.format(name)
+    utils.retry(
+        _check_volume, times=30, wait=2,
+        name='checking if volume {} exist'.format(name)
+    )
+
+
+@then(parsers.parse("the PersistentVolume '{name}' does not exists"))
+def check_pv(host, name, k8s_client):
```

Comment:
Hum, this should be similar to `check_volume_absent`, but for the PV instead (so rename it, use a `retry`, …)